content/en/blog/_posts/2021-11-12-are-you-ready-for-dockershim-removal/index.md

```diff
@@ -0,0 +1,54 @@
+Our current plan is to remove the dockershim from Kubernetes codebase soon.
+We are looking for feedback from you whether you are ready for dockershim
+removal and to ensure that you are ready when the time comes.
+**Please fill out this survey: https://forms.gle/svCJmhvTv78jGdSx8**.
+
+Dockershim component that enables Docker as a Kubernetes container runtime is
+being deprecated in favor of runtimes that directly use the [Container Runtime Interface](https://kubernetes.io/blog/2016/12/container-runtime-interface-cri-in-kubernetes/)
+created for Kubernetes. Many Kubernetes users have migrated to
+other container runtimes without problems. However we see that dockershim is
+still very popular. You may see some public numbers in recent [Container Report](https://www.datadoghq.com/container-report/#8) from DataDog.
+Some Kubernetes hosting vendors just recently enabled other runtimes support
+(especially for Windows nodes). And we know that many third party tools vendors
+are still not ready: [migrating telemetry and security agents](https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents/#telemetry-and-security-agent-vendors).
+
+At this point we believe that there is a feature parity between Docker and other
+runtimes, many customers running production workload with other runtimes, and
+[migration documentation](https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/) is clear.
+Plan of record today is that dockershim will be removed in version 1.24.
+Development of this version will start in December and the removal of dockershim
+is an irreversible operation that will happen at the beginning of the 1.24
+release cycle.
+
+So there is only one month left to give us feedback. We want you to tell us how
+ready you are.
+
+**We are collecting opinions through this survey: [https://forms.gle/svCJmhvTv78jGdSx8](https://forms.gle/svCJmhvTv78jGdSx8)**
+To better understand preparedness for the dockershim removal, our survey is
+asking the version of Kubernetes you are currently using, and an estimate of
+when you think you will adopt Kubernetes 1.24. All the aggregated information
+on dockershim removal readiness will be published.
+Free form comments will be reviewed by SIG Node leadership. If you want to
+discuss any details of migrating from dockershim, report bugs or adoption
+blockers, you can use one of the SIG Node contact options any time:
+https://github.com/kubernetes/community/tree/master/sig-node#contact
+
+Kubernetes is a mature enterprise ready product. This deprecation is another
+step in the effort to get away from permanent beta features and providing more
+stability and compatibility guarantees. With the migration from dockershim you
+will get more flexibility and choice of container runtime features as well as
+less dependencies of your apps on specific underlying technology. Please take
```

Comment:
To carry over my comment from the old doc, I an slightly worried about this last sentence as potentially reading as threatening but I am too brain fried to suggest a rewording. Anyone have suggestions?

Reply:
+1 to Rey's suggestion. That sounds way less threatening

Reply:
I rephrased a bit including the sentence about consulting with k8s hosting vendor. Thank you!